app/presenters/contacts_finder_presenter.rb

```diff
@@ -0,0 +1,101 @@
+class ContactsFinderPresenter
+
+  def initialize(contact_group)
+    @contact_group = contact_group
+    @organisation = contact_group.organisation
```

Comment:
Well since this is happening when a contact_group is updated, won't the current one always be the max?